### PR TITLE
Override extra_vars from .vagrantuser

### LINKED ISSUE #5: add swap file/partition
```
sudo dd if=/dev/zero of=/swapfile bs=1M count=1024
sudo mkswap /swapfile
sudo swapon /swapfile
```

### PR DESCRIPTION
**Pull request by venimus**
I propose this to add another level of flexability for tweaking provisioning
For instance change php version

The trick is that passing extra vars override any other defined variables in ansible (have highest priority)

### CODE REVIEW

**Comment by venimus** [comment]
@dincho review

File: README.md [hunk 1]
```diff
@@ -80,6 +82,21 @@ pagrant:
...
+
+Default configuration is optimal for common use but you are able to tweak the provisioning by passing certain vars that are used in Ansbile roles and tasks.
+You can override the values you want by defining the `extra_vars` section in `.vagrantuser`.
+
+E.g. `/cool_project/vagrant/.vagrantuser`:
+
+```yml
+pagrant:
+  extra_vars:
+    php_fpm_version: '7.2' # Change the PHP version. Default is '7.1'. Supported are 5.6, 7.0, 7.1, 7.2
```
Comment (dincho) [inline README.md]: I'm not sure this would work with 5.6

**Comment by dincho** [comment]
Not bad. Please remove 5.6 or make sure it really works, because I'm in doubt it is.

File: README.md (hunk 1 above)
Comment (venimus) [reply README.md]: it says this in the role. Didn't test it with 5.6
you might need to redefine the modules too

**Comment by venimus** [comment]
I tested it with 7.1 and 7.2, should work with 7.0 too. Also might work with 5.6 because it is written in the Role docs but I suspect that you might need to redefine the modules too

**Comment by venimus** [comment]
There is a side effect that if you just change the variable and run provision, it will not uninstall the old version then you will end up with 2 php's on the same socket and will fail to provision.

**Comment by dincho** [comment]
> There is a side effect that if you just change the variable you will end up with 2 php's on the same socket and will fail to provision.

Any ideas how can we ** easily** solve this? If not let's just document the behaviour.

**Comment by dincho** [comment]
> I tested it with 7.1 and 7.2, should work with 7.0 too. Also might work with 5.6 because it is written in the Role docs but I suspect that you might need to redefine the modules too

Then, let's remove the 5.6 from the docs example. We don't want to encourage 5.6 usage anyway.

**Comment by venimus** [comment]
> Any ideas how can we ** easily** solve this? If not let's just document the behaviour.

One of the ideas is to add the php version to the socket name (and even add it to the site names). But it is half a solution.

Else I think we just need to handle the change in some tasks that will remove the old version before adding the new one. I think this goes into different issue.


> Then, let's remove the 5.6 from the docs example. We don't want to encourage 5.6 usage anyway.

Removed.

**Pull request merged**